makiflow/layers/trainable_layers.py

```diff
@@ -260,15 +261,26 @@ def __init__(self, D, name):
             Dimension of bias vector.
         name : str
             Name of this layer.
+        trainable : bool
+            If true, bias will be learned in train, otherwise its will be constant variable.
+        b : numpy array
+            Bias' weights. This value is used for the bias initialization with pretrained bias.
         """
         self.D = D
         self.name = name
+        self.trainable = trainable
+
+        if b is None:
+            b = np.zeros(D)
+        elif b.shape[0] != D:
+            raise ValueError(f"Initial value of `b` should have same dimension size like D={D}")
 
-        b = np.zeros(D)
         params = []
         self.bias_name = f'BiasLayer_{D}' + name
-        self.b = tf.Variable(b.astype(np.float32), name=self.bias_name)
-        params = [self.b]
+        self.b = tf.Variable(b.astype(np.float32), trainable=trainable, name=self.bias_name)
```

Comment:
Delete the` trainable` parameter here since IT IS NEVER USED IN MAKIFLOW FOR VARIABLES CREATION. Therefore, it is just strange to use it here, others may get confused by this code because there is no real reason to write `trainable=trainable`.